### ISSUE TITLE
Unexpected memory growth when using refresh_interval

### ISSUE THREAD

**Issue by benjunmun** [prompt-toolkit/python-prompt-toolkit#818]
I'm working on a prompt toolkit based application that has issues with unbounded memory usage. I believe that I've narrowed the issue down to using the `refresh_interval` feature of `prompt_toolkit.prompt`.

The following sample code consumes an additional 5 MB every 10 minutes sitting idle at the prompt. My actual code with a fancier prompt seems to grow faster than this. Turning the refresh_interval to a sub-second rate increases this dramatically, making it easier to reproduce. Without refresh_interval, the prompt can sit there forever with no change in memory usage. When measuring, I'm looking at the growth of RSS with [mprof](https://pypi.org/project/memory-profiler/) and OS tools.

```
import prompt_toolkit

prompt_toolkit.prompt(
    '>',
    refresh_interval=1,
    bottom_toolbar='test',
)
```

I'm testing with Python 3.6.7, prompt-toolkit version 2.0.7 on Linux.

I'm not super familiar with prompt toolkit yet, so please let me know if I can provide any additional information or if I may be doing something incorrectly.

**Comment by benjunmun** [comment]
Looks like this was fixed by 6f36fe6e2c31328d07b06f118bf2f4209ad3dd2d. I can use the git version for now, but is there a schedule for the next release?

**Comment by shentuchuan** [comment]
My problem looks the same as yours. Have you solved your problem or have any ideas to discuss

**Comment by benjunmun** [comment]
@shentuchuan My specific problem was fixed as of the 2.0.8 release. If you're still having problems, it's probably a different cause, and might be worth opening a separate issue for that. In general, I found it safer to create a PromptSession rather than relying on the prompt() shortcut especially for areas of the application that loop repeatedly.

**Comment by shentuchuan** [comment]
@benjunmun  thanks for your advice, I have solved this problem ; the  demo is follow:
from prompt_toolkit import PromptSession
session = PromptSession()
for i in range(500):
    session.prompt(message=">")
